kickstart/scripts/samba-deploy.sh

```diff
@@ -11,6 +11,18 @@ deploy_samba_ubuntu() {
   chown nobody:nogroup /opt/data/fieldpapers
   chmod 777 /opt/data/fieldpapers
 
+  mkdir -p /opt/data/backups/omk
+  chown nobody:nogroup /opt/data/backups/omk
+  chmod 777 /opt/data/backups/omk
```

Comment:
Not here. The FP batch upload folder and public are `777` because they're intended to be read/written by everyone.

`/opt/data/backups/omk` should be owned by `admin:admin` and be `755` so that guests can read it but not delete directories (they'll be reading as `nobody:nogroup`, hence the `55` / aka `rx`).

@danielduhh can you move the creation of this directory into `admin-deploy.sh`, as it's really posm-admin-related?